core/templates/pages/library-page/library-footer/library-footer.component.html

```diff
@@ -3,3 +3,15 @@
     <oppia-footer></oppia-footer>
   </div>
 </div>
+<style>
+  @media(max-width: 844px) {
+    .oppia-footer {
+       width: 105%;
+    }
+  }
+  @media(max-width: 300px) {
+    .oppia-footer {
+       width: 106%;
+    }
+  }
```

Comment:
Hi, these changes may not be needed. Please see https://github.com/oppia/oppia/issues/11816#issuecomment-770022599.

Reply:
@kevintab95 PTAL. 
Please see #11816

Reply:
Hm, interesting. I think we set 100% width for the footer element, right? Why does it not fill the screen as it should. Can you please find out the root cause of that?

Reply:
@kevintab95 Yes I can give it a short. Thanks.

Reply:
@kevintab95 I think the width in .oppia-navbar is set to 104% that's creating a little problem. 
![tt](https://user-images.githubusercontent.com/69013647/106350740-3c215f00-62fd-11eb-9338-20d2cd1a55e2.JPG)

Now if we are setting the overflow to hidden in the .oppia-main-body the whole problem is getting resolved. But the main problem is that it is present in oppia.css and if we use it over here the whole website might change.

https://user-images.githubusercontent.com/69013647/106350968-e64db680-62fe-11eb-9028-a83bdbaaf797.mp4

Thanks.

Reply:
> @kevintab95 I think the width in .oppia-navbar is set to 104% that's creating a little problem.
> ![tt](https://user-images.githubusercontent.com/69013647/106350740-3c215f00-62fd-11eb-9338-20d2cd1a55e2.JPG)
> 
> Now if we are setting the overflow to hidden in the .oppia-main-body the whole problem is getting resolved. But the main problem is that it is present in oppia.css and if we use it over here the whole website might change.
> 
>  gg.mp4 
> Thanks.

@kevintab95 PTAL.
I have added some outcomes which I got.

Reply:
Can you check if we have this same issue in other pages where the footer is shown? If so, might be worth doing a common fix.

Reply:
@kevintab95 Yes it on the other pages too. Can I proceed to solve this in oppia. css file.

Reply:
Yep but make sure to test this in an organised manner e.g. You can create a doc with before after images for the different screen sizes in all the places where this issue exists. This is to ensure that your changes do not result in a regression.